packages/website/src/components/layout/web/primary/cta/Search.tsx

```diff
@@ -0,0 +1,166 @@
+    setHits(response.data.hits)
+  }, [])
+
+  useEffect(() => {
+    searchQuery(debouncedInputValue).catch(console.error)
+  }, [debouncedInputValue, searchQuery])
+
+  const handleInputChange = (e: ChangeEvent<HTMLInputElement>) => {
+    setInputValue(e.target.value)
+  }
+
+  const handleClear = () => {
+    setInputValue('')
+  }
+
+  const selectDefaultSnippet = (snippets: Snippet[]) => {
+    return snippets.find(c => c.matchLevel === 'full')?.value ||
+      snippets.find(c => c.matchLevel === 'partial')?.value ||
+      snippets[0]?.value || ''
+  }
+
+  const autoFocus = useCallback((inputElement: HTMLInputElement | null) => {
+    console.log('input element focus', inputElement)
+    if (inputElement) {
+      console.log('why not focusing?')
+      inputElement.focus()
+    }
+  }, [])
+
+  return (
+    <div
+      className={'sm:absolute sm:top-16 sm:left-1/2 sm:-translate-x-1/2 sm:w-3/4 md:w-3/5 lg:w-1/2 h-fit z-[51] overflow-y-hidden'}
+    >
+      <div
+        className="relative flex items-center w-full h-12 rounded-md focus-within:shadow-lg bg-white overflow-hidden"
+      >
+        <div className="grid place-items-center h-full w-12 text-gray-300">
+          <Image
+            height={24}
+            width={24}
+            src={discordIconImg as string}
+            alt={'Join the discord server'}
+            className="h-[16px] sm:h-[16px] w-[16px] sm:w-[16px]"
+          />
+        </div>
+
+        <input
+          ref={autoFocus}
+          className="peer h-full w-full outline-none text-sm text-gray-700 pr-8"
+          type="text"
+          id="search"
+          placeholder="Search something.."
+          value={inputValue}
+          onChange={handleInputChange}
+        />
+
+        {inputValue && (
+          <button
+            className="absolute right-2 top-1/2 transform -translate-y-1/2 bg-white hover:bg-gray-dark rounded-full"
+            onClick={handleClear}
+          >
+            <svg
+              className="ais-SearchBox-resetIcon"
+              viewBox="0 0 20 20"
+              width="10"
+              height="10"
+              aria-hidden="true"
+            >
+              <path
+                d="M8.114 10L.944 2.83 0 1.885 1.886 0l.943.943L10 8.113l7.17-7.17.944-.943L20 1.886l-.943.943-7.17 7.17 7.17 7.17.943.944L18.114 20l-.943-.943-7.17-7.17-7.17 7.17-.944.943L0 18.114l.943-.943L8.113 10z"
+              />
+            </svg>
+          </button>
+        )}
+
+      </div>
+
+      <div className={'bg-white rounded-sm px-8 py-4 mt-4 flex flex-col divide-dashed divide-y'}>
+        {hits.map((hit) => (
+          <Link
+            key={hit.objectID}
+            href={hit.url}
+            onClick={clicked}
+            className=" cursor-pointer text-black pt-4 pb-4"
+          >
+            <h2>
+              {hit.headingText}
+            </h2>
+            <p
+              dangerouslySetInnerHTML={{ __html: selectDefaultSnippet(hit._snippetResult?.content ?? []) }}
```

Comment:
Not possible to avoid this?

Reply:
we are using highlights that are coming back from algolia's response and it's noted with <mark> & </mark>. If we don't do this, they'll end up showing up as just text. We can create another component to hide this, but it must be done at some level.